src/request.ml

```diff
@@ -766,7 +766,7 @@ END
 value special_vars =
   ["alwsurn"; "cgl"; "dsrc"; "em"; "ei"; "ep"; "en"; "eoc"; "escache"; "et";
    "iz"; "log_cnl"; "log_pwd"; "log_uid"; "long"; "manitou"; "nz"; "ocz";
-   "pz"; "pure_xhtml"; "size"; "spouse"; "templ"]
+   "pz"; "pure_xhtml"; "size"; "spouse"; "templ"; "mod"]
```

Comment:
OK this is good, but I'm not a big fan of the variable name. It's a little confusing with other request with MOD. If it's about perso module, or perso options, I would prefer something like popt (to keep it short).

Also, you must add the necessary documentation in the gwf file because right now, we have no idea how to use it.